SofaKernel/framework/sofa/simulation/SceneLoaderFactory.h

```diff
@@ -66,14 +66,14 @@ class SOFA_SIMULATION_CORE_API SceneLoader
     virtual bool canWriteFileExtension(const char * /*extension*/) { return false; }
 
     /// load the file
-    sofa::simulation::Node::SPtr load(const char *filename)
+    sofa::simulation::Node::SPtr load(const char *filename, const std::vector<std::string>* sceneArgs = NULL)
```

Comment:
passing a const reference is often a better choice. No risk of null pointer so no corner case to handle in the code. 
The default parameter can be an empty vector of string.

Reply:
I totally agree with you @damienmarchal . I made the asked modifications. Please tell me if it is OK.